ChangeLog

```diff
@@ -23,6 +23,8 @@ Features
      may be up to two bytes shorter. This allows the library to support all
      hash and signature sizes that comply with FIPS 186-4, including SHA-512
      with a 1024-bit key.
+   * Add key_pwd command line parameter to ssl_server2 and ssl_client2
```

Comment:
Negligible remark: two arguments are added to `ssl_server`, so maybe this should be formulated more generally, without refering to `key_pwd`, and hence omitting `key_pwd2`, or more precisely, taking all the new parameters into account.

Reply:
I'll reformulate it.

Reply:
Fixed.